pipelines/upgrade/05-server_to_intermediate.yaml

```diff
@@ -21,3 +21,11 @@
     - foreman_server_repositories
     - role: foreman_installer
       foreman_installer_skip_installer: "{{ true if forklift_upgrade_version_start == forklift_upgrade_version_intermediate else false }}"
+  post_tasks:
+    - name: Run remove Pulp 2
+      shell: yes | foreman-maintain content remove-pulp2
```

Comment:
I've known about this pattern for a long time but I think this is the first time I see it in action :)